backend/src/db-utils.js

```diff
@@ -371,6 +371,7 @@ const queryActiveAccountsCountAggregatedByWeek = async () => {
       FROM transactions
       JOIN execution_outcomes ON execution_outcomes.receipt_id = transactions.converted_into_receipt_id
       WHERE execution_outcomes.status IN ('SUCCESS_VALUE', 'SUCCESS_RECEIPT_ID')
+      AND transactions.block_timestamp < ((CAST(EXTRACT(EPOCH FROM DATE_TRUNC('week', NOW())) AS bigint)) * 1000 * 1000 * 1000)
```

Comment:
This is the fix. The rest in this PR is just an alignment to use the same functions everywhere instead of my tricks with integer divisions